test/rules.json

```diff
@@ -1339,6 +1339,54 @@
       "transformers": [
         "lowercase"
       ]
+    },
+    {
+      "id": "test-marshalling",
+      "name": "Test ddwaf_object marshalling",
+      "tags": {
+        "type": "security_scanner",
+        "category": "attack_attempt"
+      },
+      "conditions": [
+        {
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.request.headers.no_cookies"
+              }
+            ],
+            "list": [
+              "marshalling"
+            ]
+          },
+          "operator": "phrase_match"
+        }
+      ],
+      "output": {
+        "event": false,
+        "keep": true,
+        "attributes": {
+          "_dd.appsec.trace.string": {
+            "value": "Lorem ipsum sit amet"
```

Comment:
```suggestion
            "value": "Lorem ipsum dolor sit amet"
```

Reply:
Lorem ipsum is bored, let's try something new

Reply:
_...et en conséquence, n'envoie jamais demander pour qui sonne le glas; il sonne pour toi._

Reply:
If i were you I wouldn't be shy of quotting in the language you desire, catalan or spanish, or any other. It's what make a codebase feel human :)